src/httpserver.cpp

```diff
@@ -137,16 +137,20 @@ struct HTTPPathHandler
 
 //! libevent event loop
 static struct event_base* eventBase = nullptr;
-//! HTTP server
-struct evhttp* eventHTTP = nullptr;
+namespace {
+    //! HTTP server
+    struct evhttp* eventHTTP = nullptr;
+}
 //! List of subnets to allow RPC connections from
 static std::vector<CSubNet> rpc_allow_subnets;
 //! Work queue for handling longer requests off the event loop thread
 static WorkQueue<HTTPClosure>* workQueue = nullptr;
-//! Handlers for (sub)paths
-std::vector<HTTPPathHandler> pathHandlers;
-//! Bound listening sockets
-std::vector<evhttp_bound_socket *> boundSockets;
+namespace {
+    //! Handlers for (sub)paths
+    std::vector<HTTPPathHandler> pathHandlers;
+    //! Bound listening sockets
+    std::vector<evhttp_bound_socket *> boundSockets;
+}
```

Comment:
Why can't you wrap everything above into a namespace? Also should not change the indentation. I'd suggest to use clang-format

Reply:
Good point! Fixed!

With this change `HTTPPathHandler`, `WorkQueue` and `HTTPWorkItem` are no longer global.

Please re-review!